examples/Training/matlab/DeleteData.m

```diff
@@ -64,12 +66,21 @@
     % Retrieve the roi linked to an image.
     fprintf(1, 'Reading ROIs attached to image %g\n', imageId);
     roiResult =  session.getRoiService().findByImage(imageId, []);
-    rois = roiResult.rois;
-    nRois = rois.size;
-    fprintf(1, 'Found %g ROI(s)\n', nRois);
+    rois = toMatlabList(roiResult.rois);
+    fprintf(1, 'Found %g ROI(s)\n', numel(rois));
     
+    % Remove all the shapes from the first ROI
+    roi = rois(1);
+    shapes = roi.copyShapes();
+    if (shapes.size > 0)
+        fprintf(1, '  Removing %g shapes\n', shapes.size);
+        for j = 1 : shapes.size
+            roi.removeShape(shapes.get(j-1));
+        end
+        roi = session.getUpdateService().saveAndReturnObject(roi);
+    end
+        
     % Delete ROI
-    roi = rois.get(0);
     fprintf(1, 'Deleting ROI %g\n', roi.getId().getValue());
     deleteCommand = omero.cmd.Delete('/Roi', roi.getId().getValue(), []);

```

Comment:
Should this get updated to omero.cmd.Delete2 as well (given that Delete is a deprecated method now)